packages/next/next-devtools.webpack-config.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path')
+const webpack = require('@rspack/core')
+const MODERN_BROWSERSLIST_TARGET = require('./src/shared/lib/modern-browserslist-target')
+const DevToolsIgnoreListPlugin = require('./webpack-plugins/devtools-ignore-list-plugin')
+
+function shouldIgnorePath(modulePath) {
+  // For consumers, everything will be considered 3rd party dependency if they use
+  // the bundles we produce here.
+  // In other words, this is all library code and should therefore be ignored.
+  return true
+}
+
+/**
+ * @param {Object} options
+ * @param {boolean} options.dev
+ * @param {Partial<webpack.Configuration>} options.rest
+ * @returns {webpack.Configuration}
+ */
+module.exports = ({ dev, ...rest }) => {
+  const experimental = false
+  const turbo = false
+
+  const bundledReactChannel = experimental ? '-experimental' : ''
+
+  const target = `browserslist:${MODERN_BROWSERSLIST_TARGET.join(', ')}`
+
+  return {
+    entry: path.join(
+      __dirname,
+      'src/client/components/react-dev-overlay/app/entrypoint.js'
+    ),
+    target,
+    mode: dev ? 'development' : 'production',
+    output: {
+      path: path.join(__dirname, 'dist/compiled/next-devtools'),
+      filename: `index.js`,
+      iife: false,
+      library: {
+        type: 'commonjs-static',
```

Comment:
should we try `modern-module` to get better bundle size? webpack supports it in 5.93.0+

Reply:
You can experiment with that if you want. I don't want to tinker with this for Turbopack.